postman/schemas/transfer.yml

```diff
@@ -2705,6 +2747,13 @@ components:
           type: array
           items:
             type: string
+    transferProgressResponse:
+      type: object
+      properties:
+        transferId:
+          type: string
+        status:
+          type: string
```

Comment:
Should status here be an enum with a specific set of values? If not, we will want to include all the possible values in the description section for this endpoint.

Reply:
These are interesting statuses....we might want to add a description field here explaining what each status means and what I as a consumer should do about it.

Reply:
Looks good. If you can include those in the description of the spec, I will add them to the dev docs as well